### ISSUE TITLE
Module fails to build with librdkafka v0.9.3

### ISSUE THREAD

**Issue by kinghuang** [confluentinc/confluent-kafka-python#115]
The C portions of confluent-kafka-python are failing to build with [librdkafka v0.9.3](https://github.com/edenhill/librdkafka/releases/tag/v0.9.3) (released January 23). Builds work fine with librdkafka v0.9.2.

```
Installing collected packages: argparse, six, linecache2, traceback2, unittest2, confluent-kafka
  Running setup.py develop for confluent-kafka
    Complete output from command /usr/local/bin/python3.6 -c "import setuptools, tokenize;__file__='/src/confluent-kafka/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" develop --no-deps:
    running develop
    running egg_info
    writing confluent_kafka.egg-info/PKG-INFO
    writing dependency_links to confluent_kafka.egg-info/dependency_links.txt
    writing requirements to confluent_kafka.egg-info/requires.txt
    writing top-level names to confluent_kafka.egg-info/top_level.txt
    warning: manifest_maker: standard file '-c' not found
    
    reading manifest file 'confluent_kafka.egg-info/SOURCES.txt'
    reading manifest template 'MANIFEST.in'
    writing manifest file 'confluent_kafka.egg-info/SOURCES.txt'
    running build_ext
    building 'confluent_kafka.cimpl' extension
    gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/usr/local/include/python3.6m -c confluent_kafka/src/confluent_kafka.c -o build/temp.linux-x86_64-3.6/confluent_kafka/src/confluent_kafka.o
    confluent_kafka/src/confluent_kafka.c: In function 'c_parts_to_py':
    confluent_kafka/src/confluent_kafka.c:829:17: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
      for (i = 0 ; i < c_parts->cnt ; i++) {
                     ^
    confluent_kafka/src/confluent_kafka.c: In function 'py_to_c_parts':
    confluent_kafka/src/confluent_kafka.c:858:17: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]
      for (i = 0 ; i < PyList_Size(plist) ; i++) {
                     ^
    gcc -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/usr/local/include/python3.6m -c confluent_kafka/src/Producer.c -o build/temp.linux-x86_64-3.6/confluent_kafka/src/Producer.o
    confluent_kafka/src/Producer.c: In function 'Producer_producev':
    confluent_kafka/src/Producer.c:253:16: warning: implicit declaration of function 'rd_kafka_producev' [-Wimplicit-function-declaration]
             return rd_kafka_producev(self->rk,
                    ^
    confluent_kafka/src/Producer.c:254:34: warning: implicit declaration of function 'RD_KAFKA_V_MSGFLAGS' [-Wimplicit-function-declaration]
                                      RD_KAFKA_V_MSGFLAGS(RD_KAFKA_MSG_F_COPY),
                                      ^
    confluent_kafka/src/Producer.c:255:34: warning: implicit declaration of function 'RD_KAFKA_V_TOPIC' [-Wimplicit-function-declaration]
                                      RD_KAFKA_V_TOPIC(topic),
                                      ^
    confluent_kafka/src/Producer.c:256:34: warning: implicit declaration of function 'RD_KAFKA_V_PARTITION' [-Wimplicit-function-declaration]
                                      RD_KAFKA_V_PARTITION(partition),
                                      ^
    confluent_kafka/src/Producer.c:257:34: warning: implicit declaration of function 'RD_KAFKA_V_KEY' [-Wimplicit-function-declaration]
                                      RD_KAFKA_V_KEY(key, (size_t)key_len),
                                      ^
    confluent_kafka/src/Producer.c:258:34: warning: implicit declaration of function 'RD_KAFKA_V_VALUE' [-Wimplicit-function-declaration]
                                      RD_KAFKA_V_VALUE((void *)value,
                                      ^
    confluent_kafka/src/Producer.c:260:34: warning: implicit declaration of function 'RD_KAFKA_V_TIMESTAMP' [-Wimplicit-function-declaration]
                                      RD_KAFKA_V_TIMESTAMP(timestamp),
                                      ^
    confluent_kafka/src/Producer.c:261:34: warning: implicit declaration of function 'RD_KAFKA_V_OPAQUE' [-Wimplicit-function-declaration]
                                      RD_KAFKA_V_OPAQUE(opaque),
                                      ^
    confluent_kafka/src/Producer.c:262:34: error: 'RD_KAFKA_V_END' undeclared (first use in this function)
                                      RD_KAFKA_V_END);
                                      ^
    confluent_kafka/src/Producer.c:262:34: note: each undeclared identifier is reported only once for each function it appears in
    confluent_kafka/src/Producer.c:263:1: warning: control reaches end of non-void function [-Wreturn-type]
     }
     ^
    error: command 'gcc' failed with exit status 1
    
    ----------------------------------------
Command "/usr/local/bin/python3.6 -c "import setuptools, tokenize;__file__='/src/confluent-kafka/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" develop --no-deps" failed with error code 1 in /src/confluent-kafka/
```

**Comment by ewencp** [comment]
@edenhill It looks like this might be an issue with the tagged release -- v0.9.3 doesn't include the `producev` patch. It's merged into master and is on 0.9.2.x, but not in v0.9.3.

**Comment by edenhill** [comment]
producev() never made it into 0.9.3, but the the Python module still checks for 0.9.3 to enable it.
Will fix

**Comment by edenhill** [comment]
Fixed on master, please give it a try.

**Comment by kinghuang** [comment]
Working for me. Tested with def5680. Thanks!
